release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -448,6 +448,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
 
+[id="ocp-4-19-installation-and-update-aws-malaysia-thailand{context}"]
+==== Installing a cluster on {aws-first} in the Malaysia and Thailand regions
```

Comment:
Not required, but FYI this is allowed since it's spelled out immediately after in the text. In case you want a shorter title :)
```suggestion
==== Installing a cluster on {aws-short} in the Malaysia and Thailand regions
```